src/libertem/analysis/masks.py

```diff
@@ -132,6 +135,10 @@ class SingleMaskResultSet(AnalysisResultSet):
         Sum of the selected region for each detector frame, with shape of
         the navigation dimension. Absolute of the result if the dataset or mask contains
         complex numbers.
+    intensity_log : libertem.analysis.base.AnalysisResult
+        Log-scaled sum of the selected region for each detector frame, with shape of
+        the navigation dimension. Absolute of the result if the dataset or mask contains
+        complex numbers. Added in 0.6.0.dev0
```

Comment:
Maybe `.. versionadded` should be used instead? Should make the presentation a bit more consistent, and also makes it greppable

Reply:
`..versionadded::`doesn't work in the Parameters section since that uses special formatting. For that reason I already didn't use it in other instances, for example the docstring of `create_com_analysis()` The only way I found working was placing it in the description text above, like this. What is the better option in your opinion?

![image](https://user-images.githubusercontent.com/25689052/102320013-0823f180-3f7c-11eb-9ed6-34092058d3a5.png)

Reply:
I would prefer the version using `versionadded`. Actually, it should be possible to use `versionadded`/`versionchanged` in parameters, it just needs to be indented further. See this example:

https://github.com/numpy/numpy/blob/8e038f4a930c0c899e599ee58a56391e026a608f/numpy/f2py/__init__.py#L29-L63

And how it is rendered:

https://numpy.org/doc/stable/f2py/usage.html#numpy.f2py.compile

If that doesn't work, I would still prefer having `versionadded` outside the parameters section. Maybe we can make this consistent in our next doc review.

Reply:
Oh, Thank you! It seems to need the empty lines before and after. I'll fix the instances that I know of here.

> Maybe we can make this consistent in our next doc review.

Yes, sounds good! :-)